### PR TITLE
feat(ui): allow to enter group name manually for project

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17093.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Review by yesnault — APPROVED** [review]

File: ui/src/app/shared/permission/form/workflow-permission.form.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            <label>{{'group_name' | translate}}</label>
+            <sm-select class="fluid search" [options]="{'fullTextSearch': true}"
+                [(model)]="newGroupPermission.group.name">
+                <option *ngFor="let g of groups" [value]="g.group.name">{{g.group.name}}{{g.group.organization ? ' (' +
+                    g.group.organization + ')' : ''}}</option>
+            </sm-select>
+        </div>
+        <div class="five wide field">
+            <label>{{'group_permission' | translate}}</label>
+            <sm-select class="fluid search" [options]="{'fullTextSearch': true}"
```
Comment (sguiheux) [inline ui/src/app/shared/permission/form/workflow-permission.form.html:13]: sui-select ?

**Review by sguiheux — CHANGES_REQUESTED** [review]

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17102.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17104.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

**Pull request merged**